src/Umbraco.Web.Common/ImageProviders/WebRootImageProvider.cs

```diff
@@ -0,0 +1,20 @@
+using Microsoft.AspNetCore.Hosting;
+using SixLabors.ImageSharp.Web;
+
+namespace Umbraco.Cms.Web.Common.ImageProviders
+{
+    /// <summary>
+    /// Returns images from the web root file provider.
+    /// </summary>
+    public sealed class WebRootImageProvider : FileProviderImageProvider
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="WebRootImageProvider"/> class.
+        /// </summary>
+        /// <param name="environment">The web host environment.</param>
+        /// <param name="formatUtilities">Contains various format helper methods based on the current configuration.</param>
+        public WebRootImageProvider(IWebHostEnvironment environment, FormatUtilities formatUtilities)
+            : base(environment?.WebRootFileProvider, formatUtilities)
```

Comment:
Can the `IWebHostEnvironment ` be null? `WebRootFileProvider` definitely but I would have thought the entire application would fail without the environment no?

Reply:
You're totally right, this can simply be updated to:
```suggestion
            : base(environment.WebRootFileProvider, formatUtilities)
```